### PR TITLE
Bump Marathon to 1.3.5 and test without ZooKeeper

### PR DESCRIPTION
**Pull request by rveznaver**

### CODE REVIEW

File: metadata.rb [hunk 1]
```diff
@@ -4,15 +4,15 @@
...
   supports os
 end
 
 # Cookbook dependencies
-%w( java apt yum mesos poise-service ).each do |cb|
+%w(java apt yum mesos poise-service).each do |cb|
   depends cb
 end
-
-recommends 'zookeeper'
```
Comment (pierrecdn) [inline metadata.rb]: OPT: keep `suggests` instead ?

File: .kitchen.yml [hunk 2]
```diff
@@ -1,33 +1,30 @@
...
+      - recipe[mesos::master]
+      - recipe[mesos::slave]
+      - recipe[marathon]
+      - recipe[marathon::service]
+    attributes:
+      marathon:
+        flags:  # run Marathon without ZooKeeper
+          ha: false
+          master: 'localhost:5050'
+          internal_store_backend: 'mem'
```
Comment (pierrecdn) [inline .kitchen.yml]: Nice one, which is hidden !

**Review by pierrecdn — APPROVED** [review]
Nice one, thanks !

File: metadata.rb (hunk 1 above)
Comment (rveznaver) [inline metadata.rb]: FoodCritic complains :(
http://www.foodcritic.io/#FC052
http://www.foodcritic.io/#FC053

File: .kitchen.yml (hunk 2 above)
Comment (rveznaver) [inline .kitchen.yml]: All credit goes to https://github.com/mesosphere/marathon/issues/2950#issuecomment-254235691

**Pull request merged**